### ISSUE TITLE
Laravel 9 support?

### ISSUE THREAD

**Issue by eugenebednik** [stidges/laravel-country-flags#6]
Hi there,

Will there be Laravel 9.x support for this package?

I am getting composer conflicts while installing this:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires stidges/laravel-country-flags ^2.0 -> satisfiable by stidges/laravel-country-flags[v2.0.0].
    - stidges/laravel-country-flags v2.0.0 requires illuminate/support ^7.0|^8.0 -> found illuminate/support[v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.

You can also try re-running composer require with an explicit version constraint, e.g. "composer require stidges/laravel-country-flags:*" to figure out if any version is installable, or "composer require stidges/laravel-country-flags:^2.1" if you know which you need.
```

Thaks!

**Comment by RonJeremyR6** [comment]
Yeah, got the same problem at the moment.

**Comment by stidges** [comment]
Closed by #8 thanks to @toomasr :)
